src/components/settings/SettingsLMSTab/tests/LmsConfigPage.test.jsx

```diff
@@ -217,6 +225,33 @@ describe('<SettingsLMSTab />', () => {
     expect(screen.queryByText('Exit without saving')).toBeFalsy();
     expect(screen.queryByText('Connect SAP')).toBeFalsy();
   });
+  test('Expected behavior when customer has no IDP configured', async () => {
+    const expectedApiResponse = {
+      data: { needs_idp_config: true },
+    };
+    mockNeedsIDPConfig.mockImplementation(() => (Promise.resolve(expectedApiResponse)));
+    const history = createMemoryHistory();
+    render(
+      <Router history={history}>
+        <Provider store={mockStore({ ...initialState })}>
+          <SettingsLMSTab enterpriseId={enterpriseId} enterpriseSlug={enterpriseSlug} />
+        </Provider>
+      </Router>,
+    );
+    expect(history.location.pathname).toEqual('/');
+    await screen.findByText('Configure SSO');
+    const configureSSOButton = screen.getByText('Configure SSO');
+    fireEvent.click(configureSSOButton);
```

Comment:
please do ensure you are not getting the act wranings in these tests, I had to use `findByText` and await using act to avoid those warnings, 

here is a PR for your ref, if that is happening at all for you: 

https://github.com/openedx/frontend-app-enterprise-public-catalog/pull/199

Reply:
I'm finding no act warnings as I run tests!